### ISSUE TITLE
Custom timestampFormat

### ISSUE THREAD

**Issue by klemensz** [dbfannin/ngx-logger#178]
Hi,
I would like to specify a format like `HH:mm:ss.SSS`, but the TypeScript compiler will complain, because it's not one of the predefined formats like 'short', 'medium', 'long', etc. Is it possible to add the option to provide a custom format (like we can with the Angular date pipe)? 

BTW, this will work, but it's not a very clean solution:

```ts
        LoggerModule.forRoot({
            serverLogLevel: NgxLoggerLevel.OFF,
            // @ts-ignore
            timestampFormat: 'HH:mm:ss.SSS'
        })
```
Thanks!

**Comment by MicMicMon** [comment]
You can use all standard formats that are understood by the built-in Angular date pipe (listed [here](https://angular.io/api/common/DatePipe)).

This choice has been made to avoid pulling a new dependency just for formatting dates.

**Comment by klemensz** [comment]
I know I can (see my text above), but it requires a @ts-ignore rule to suppress the error.

**Comment by MicMicMon** [comment]
Ah I see, sorry. What is the error message?

**Comment by klemensz** [comment]
Error message: 

`Type ""HH:mm:ss.SSS"" is not assignable to type ""short" | "medium" | "long" | "full" | "shortDate" | "mediumDate" | "longDate" | "fullDate" | "shortTime" | "mediumTime" | "longTime" | "fullTime"". ts(2322)`

**Comment by MicMicMon** [comment]
Aw right! I didn't see the custom options when I did this PR. The type shouldn't be restricted and all strings should be allowed.

I will submit a PR to fix it, sorry about that.

**Comment by qortex** [comment]
#186 fixes the issue and a patch will be published soon.

**Comment by qortex** [comment]
Now fixed in published 4.1.5. Sorry for the delay in the release!

**Comment by klemensz** [comment]
Thanks.
